cea/plots/demand/energy_balance.py

```diff
@@ -159,25 +158,25 @@ def calc_graph(analysis_fields, data_frame):
     return graph
 
 
-
-
-
 def calc_monthly_energy_balance(data_frame, normalize_value):
     """
     calculates heat flux balance for buildings on hourly basis
 
-    :param data_frame:
+    :param data_frame: demand information of building in pd.DataFrame
+    :param normalize_value: value for normalization of thermal energy fluxes, usually GFA
     :return:
     """
 
+    # invert sign of I_rad, I_rad was previously negative but it was changed to be consistent with ISO 13790
+    data_frame['I_rad_kWh'] = -data_frame['I_rad_kWh']
+
     # calculate losses of heating and cooling system in data frame and adjust signs
     data_frame['Qhs_loss_sen_kWh'] = -abs(data_frame['Qhs_em_ls_kWh'] + data_frame['Qhs_dis_ls_kWh'])
     data_frame['Qhs_tot_sen_kWh'] = data_frame['Qhs_sen_sys_kWh'] + abs(data_frame['Qhs_loss_sen_kWh'])
     data_frame['Qcs_loss_sen_kWh'] = -data_frame['Qcs_em_ls_kWh'] - data_frame['Qcs_dis_ls_kWh']
     data_frame['Qcs_tot_sen_kWh'] = data_frame['Qcs_sen_sys_kWh'] - abs(data_frame['Qcs_loss_sen_kWh'])
-    data_frame['Qcs_tot_lat_kWh'] = -data_frame['Qcs_lat_sys_kWh']  # change sign because this variable is now positive in demand
-
-
+    # change sign because this variable is now positive in demand
```

Comment:
here again "is now positive in demand" refers to historical context of the code (i'm guessing) and i think that adds a dimension we shouldn't be adding. a comment describing _why_ the value needs to be changed (just for the plot, right?) would be interesting. (also, the totals is just the system stuff?)